adr/00005-offline-security.md

```diff
@@ -0,0 +1,40 @@
+# 5. Offline security
+
+Date: 2023-05-04
+
+## Status
+
+Accepted
```

Comment:
Would it be worth saying that this is in-progress as we're still deciding on the exact mechanism for the encryption key? Or is that an addendum, or follow-up ADR?

Reply:
We totally can do this.